history.txt

```diff
@@ -19,6 +19,7 @@ Improvements include:
 -  increased OMERO.web log size
 -  fixed CLI config list subcommand
 -  fixed leaking services in OMERO.py
+-  improved rendering of large images (non tiled) using OMERO.py and webgateway
```

Comment:
"large images (non tiled)" -> "non-tiled large images"

Otherwise 👍 -I'll add to the announcement